layouts/partials/default.html

```diff
@@ -22,7 +22,27 @@
           {{ partial "breadcrumb.html" . }}
         </div>
         <!-- end breadcrumb feature -->
-        <div class="p-lg-5 p-4 bg-white" id="content">
+
+        <!-- begin "Edit This Page" button -->
+        <div class="p-lg-5 p-4 bg-white position-relative" id="content">
+          <div class="d-flex flex-row-reverse">
+            {{ $path := "" }}
+            {{ with .File }}
+                {{ $path = .Path }}
+                {{ else }}
+                {{ $path = .Path }}
+            {{ end }}
+            {{ if eq .Params.hide_edit_btn true }}
+                {{ else }}
+                <a href="{{ .Site.Params.editPrefix}}/{{ $path }}">
+                  <button class="editBtn py-2 px-3"><span>Edit this Page
+                    </span></button>
+                </a>
+            {{ end }}
+          </div>
+        <!-- end "Edit This Page" button -->
+
+        <div class="p-lg-5 p-3 bg-white" id="content">
```

Comment:
This line seems to be misplaced. Note the Bootstrap class change from `p-4` to `p-3` and the same `id` value as line 27. I'm not sure which one is correct or if the change was accidental.

Reply:
It was accidental. I removed it🤞